### DISCUSSION TITLE
Troubles with custom tuning

### DISCUSSION THREAD

**Discussion by Wilhelm-von-Seitz** [JamesCJ60/Universal-x86-Tuning-Utility#219] [General]
Good afternoon. I have a problem with understanding of how custom tuning works. No matter how I try to change the values, there is no noticeable effect. Essentially, only the existing presets work. The best result is in extreme mode, but even that is not the limit. I know that I can get more out of my processor, but nothing is working. The 7840HS is capable of maintaining stable 4.8 GHz on all cores under full load without exceeding thermal limits. There is no guide on how to manually configure the utility. This utility is basically only known software that can help people with laptop CPUs. I'm grateful for what we have 👍🏻 thanks 🙏🏻

**Comment by JamVzxc** [comment]
I have troubling trying to custom tune my HP omen 16 gaming laptop with AMD Ryze 7 7840HS w Radeon 780M as well
